src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -1347,15 +1372,26 @@ private function getServiceCall($id, Reference $reference = null)
      *
      * @throws InvalidArgumentException
      */
-    private function camelize($id)
+    private function generateMethodName($id)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid variables with short names like $id. Configured minimum length is 3.](https://www.codacy.com/app/git_9/symfony/file/2331537296/issues/source?bid=3207821&fileBranchId=3207862#l1375)